Create page in AMPLIFY Central section “manage-environments-ownership” (#1803)
Axway/axway-open-docs: content/en/docs/central/manage-environments-ownership.md

---

> title: Assign an environment owner

Reviewer: Environment has the first letter in uppercase in some places, but in some is lower case. Can this be changed to be consistent everywhere?  
Examples: "Learn how to limit access to Environments and API Services" vs "limit access to API Services in that environments".

description: Learn how to limit access to Environments and API Services by
  assigning an owning team.
---

> Within Amplify Central you can configure an environment's ownership to limit access to API Services in that environments. By assigning an owner, Administrators can delegate full control over the API services within that environment to the members of the owning team, as further indicated by their user role. Members outside of the owning team will not be able to view or create API Services in that environment.

Reviewer: First sentence does not sound that good to me :). Can that be rephrased?

#### Who can configure the environment ownership?

> Users that are assigned the Platform Administrator role in combination with Central Administrator role can assign an owning team, either when creating the environment or after creation.

Reviewer 2: ```suggestion
Users that are assigned the Platform Administrator role in combination with the Central Administrator role can assign an owning team, either when creating the environment or after creation.
```

> Users that are assigned the Platform Administrator role in combination with the Central Administrator role can assign an owning team, either when creating the environment or after creation.

Reviewer 3: "assign an owning team when creating the environment or change the owning team after creation"

#### Environment ownership

> Setting up an owner for an environment is OPTIONAL. You can create one without configuring the ownership. That means only Central Administrators can view and manage that environment and the API Services within that scope. No other members in the organization will be able to view or access those resources, until an owner is assigned.

Reviewer 2: ```suggestion
Environments can be created without configuring the ownership so setting an owner is OPTIONAL. That means only Central Administrators can view and manage that environment and the API Services within that scope. No other members in the organization will be able to view or access those resources, until an owner is assigned.
```

#### API Service ownership

> When creating an API Service, you can assign an owner as well. However, the API Service must have the same owner as the environment the service was registered under. If no owner is assigned, then that API Service will inherit the environment's owner. The same rules apply for the rest of the resources.

Reviewer 3: "The same rules apply to all the resources created within the environment"

#### How to assign an owner?

Currently, you can assign an owner by using the Amplify Central CLI. To learn how to create an environment using the CLI, see [Build an environment](https://docs.axway.com/bundle/axway-open-docs/page/docs/central/cli_central/cli_environments/index.html).

Below you can see a sample of an environment that has an owning team.

```json
{
  "apiVersion": "v1alpha1",
  "group": "management",
  "title": "Json env title one",
  "name": "env1",
  "kind": "Environment",
  "attributes": {
    "createdBy": "json",
    "randomNum": "1"
  },
  "tags": [
    "cli",
    "axway"
  ],
 "owner":{
    "type":"team",
    "id":"7a0c3957-8df5-4fdc-8ecd-df8240c1233a"
  },
  "spec": {
    "description": "spec description one"
  }
 }
```

When configuring the owner, you need to specify the following properties:

* **type**: the type of the owner. It defaults to "team", if not present. Currently, only the \`team\` type is supported.

> * **id:** the id of the owner. When the type is set to \`team\`, set the value to the id of the team you want to assign as owner.

Reviewer 3: Should we include some text around "if the id is not specified or set to double quotes in the json file, the ownership will be assigned to the environment owner where the API Service is created"

Author: That example is for an Environment. I can add one for assigning an owner to an API Service and add the text that you suggested.
